decrypt.go

```diff
@@ -78,27 +78,26 @@ func (ds *decryptStream) read(b []byte) (n int, err error) {
 		n, last, ds.err = ds.readBlock(b)
 		if ds.err != nil {
 			return 0, ds.err
```

Comment:
Moot, but I agree (although readBlock returns a non-nil err exactly when it returns 0).